bezier-rs/docs/interactive-docs/src/App.vue

```diff
@@ -34,6 +34,9 @@ const testBezierLib = async () => {
 	});
 };
 
+const NON_INTERACTIVE_STROKE_1 = "red";
```

Comment:
It might be nice to change this to the following and then move it to the `drawing.ts` file:
```
const COLORS = {
  INTERACTIVE: {
    STROKE_1: "black",
    STROKE_2: "grey",
    SELECTED: "blue",
  },
  NON_INTERACTIVE: {
    STROKE_1: "red",
    STROKE_2: "orange",
  }
}
```
Using the word "color" here might be more insightful than "stroke" on its own.
Also, maybe we can use the color "grey" for the line draw from a point to its handler?
If you use this, make sure to replace instances of `"black"` and `"blue"` to use the new constant!